docs/Tutorials/Examples/Private-Network-Example.md

```diff
@@ -238,169 +252,172 @@ created during the genesis of this private test network.
     [account management](../../HowTo/Send-Transactions/Account-Management.md). To create your own
     account, you have to use a third-party tool, such as MetaMask.
 
-After you sign in to MetaMask, connect to the private network RPC endpoint:
+When you're finished importing an existing test account, [create another test account from scratch]
+to use as the recipient for a test Ether transaction.
```

Comment:
MetaMask install comes with a default account ("Account 1"). This could be also used as a test Ether recipient - but not completely necessary to add to tutorial.

Reply:
The risk is that user already uses it for something else on another network and it's dangerous to have an account used for test and production at the same time.